### DISCUSSION TITLE
Jeasx 1.0 released - a light-weight web-framework on top of Fastify which combines the ease of JSX with the power of SSR

### DISCUSSION THREAD

**Discussion by jablonski** [fastify/fastify#5733] [Show and tell]
Hello to all,

I've just released Jeasx 1.0 which blends Fastify 5 with the server-side rendering capabilities of asynchronous JSX. 

Jeasx embraces modern HTML and CSS as the core building blocks for web development. It provides the flexibility to seamlessly enhance functionality using JavaScript, empowering developers to create dynamic web sites with minimal resource consumption.

Thanks to the fantastic work done by the Fastify community, Jeasx itself is just about 100 lines of code, which makes the framework very robust and maintainable.

[https://www.jeasx.dev/](https://www.jeasx.dev/)

Cheers,
Maik

<img width="1284" alt="Screenshot 2024-10-07 at 10 14 03" src="https://github.com/user-attachments/assets/4e2954fe-c55b-4f85-9b9d-04e5790f4e49">

**Comment by mcollina** [comment]
Nice job! Would you like to add it to our ecosystem page?

**Reply by jablonski** [reply]
Yes, that would be nice, but at the moment the Fastify-Ecosystem-Page lists only actual Fastify-plugins.

Jeasx is intended as standalone framework with all required "Fastify batteries" already included (and not available as plugin).

Maybe the Ecosystem-Page can be extended for Fastify-powered projects or something similar?

**Reply by mcollina** [reply]
Send a PR maybe?

**Reply by jablonski** [reply]
I'll take a closer look at this at the weekend.
